lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -0,0 +1,83 @@
+module Importers
+  module LicenceTransaction
+    class LicenceImporter
+      def call
+        filtered_licences.each do |licence|
+          details = licence["details"]
+          new_licence = ::LicenceTransaction.new(
+            locale: "en",
+            base_path: "/find-licences#{licence['base_path']}",
+            title: licence["title"],
+            summary: licence["description"],
+            body: details["licence_overview"].first["content"],
+            publication_state: "draft",
```

Comment:
Minor, but shouldn't this match the publication state from publishing-api? Especially as only published licences are being retrieved.

Reply:
No as we need to create a draft first before we can publish, as we can't just publish a newly created document without their being a draft (put_content and then publish). The results is only one published edition which is the correct state.